### ISSUE TITLE
Library X has no include paths

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/compiler-explorer#1682]
Seen when starting up staging:

```
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library boost 164 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library boost 165 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library boost 166 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library boost 167 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library boost 168 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library boost 169 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library boost 170 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library boost 171 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library brigand trunk (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library brigand 130 (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library kvasir trunk (c++) has no include paths 
Nov 15 07:42:25 ip-172-30-5-138 amazon.staging warn: Library cmcstl2 trunk (c++) has no include paths
```

**Comment by partouf** [comment]
https://github.com/mattgodbolt/compiler-explorer/blob/f53ad05442d41699a1a6a2d079b1d6fc8ed9876f/lib/options-handler.js#L203

when no libpath is set, which is everything except for openssl and cs, removing warning altogether probably best

**Comment by RabsRincon** [comment]
Closed by 29c863fa0786910c72273dc4952296e6fdd482b9
